Extensions/tf2_reblogs.js

```diff
@@ -0,0 +1,34 @@
+//* TITLE TF2 Reblogs **//
+//* VERSION 1.0.0 **//
```

Comment:
version should be bumped to 1.0.1 so anyone who installed this from pastebin can get the changes we make here automatically